source/options.ts

```diff
@@ -68,16 +75,22 @@ export const buildOptions = (helpText, options) => {
 		normalize: false,
 	});
 
-	const parsedOptions = {
-		pkg: foundPackage ? foundPackage.packageJson : {},
+	const pkg = foundPackage?.packageJson ?? {};
+	normalizePackageData(pkg);
+
+	const parsedOptions: ParsedOptions = {
+		pkg,
 		argv: process.argv.slice(2),
 		flags: {},
 		inferType: false,
 		input: 'string',
-		help: helpText,
+		description: pkg.description ?? false,
+		help: helpMessage,
+		version: pkg.version ?? '', // TODO: maybe say "No version found"?
```

Comment:
Should we add a fallback for the version message? Currently, without `options.version` and `package.json.version`, the result is empty (tested by removing `?? ''`):

```sh
foo-cli/
$ foo --version

foo-cli/
$
```

Reply:
Yeah, it can be `No version found`.